src/core/credentials/transport/tls/spiffe_utils.cc

```diff
@@ -0,0 +1,158 @@
+//
+//
+// Copyright 2025 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+#include "src/core/credentials/transport/tls/spiffe_utils.h"
+
+#include <string>
+
+#include "absl/strings/match.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_format.h"
+#include "absl/strings/str_split.h"
+
+constexpr absl::string_view SPIFFE_PREFIX = "spiffe://";
```

Comment:
nit: I believe this is the old style in gRPC for constants, but (I believe) new code is following the google3 style, so you'd call the variable `kSpiffePrefix`.

Reply:
Done